frontend/src/components/pages/AdminDashboard/AdminCreatorProfiles.tsx

```diff
@@ -0,0 +1,334 @@
+          },
+        },
+        MuiTableFooter: {
+          root: {
+            "& .MuiToolbar-root": {
+              backgroundColor: "white",
+            },
+          },
+        },
+      },
+    });
+
+  const getTableColumns = (): MUIDataTableColumn[] => {
+    const columns: MUIDataTableColumn[] = [
+      {
+        name: "id",
+        label: "Id",
+        options: {
+          display: false,
+        },
+      },
+      {
+        name: "name",
+        label: "Creator",
+        options: {
+          setCellProps: () => ({ style: { minWidth: "220px" } }),
+        },
+      },
+      {
+        name: "email",
+        label: "Email",
+        options: {
+          setCellProps: () => ({ style: { minWidth: "220px" } }),
+        },
+      },
+      {
+        name: "createdAt",
+        label: "Date Created",
+        options: {},
+      },
+      {
+        name: "isApproved",
+        label: "Is Approved",
+        options: {
+          display: false,
+        },
+      },
+      {
+        name: "actions",
+        label: "Actions",
+        options: {
+          // eslint-disable-next-line
+          filter: false,
+          customBodyRender: (value, tableMeta, updateValue) => {
+            return (
+              <div>
+                {tableMeta.rowData[4] ? (
+                  <>
+                    <Tooltip label="Edit review">
+                      <IconButton
+                        aria-label="edit creator"
+                        icon={<EditIcon color="#718096" />}
+                        onClick={() => {}}
+                      />
+                    </Tooltip>
+                    <Tooltip label="Delete">
+                      <IconButton
+                        aria-label="delete"
+                        icon={<DeleteIcon color="#718096" />}
+                        onClick={() => {
+                          setIsDeleteModalOpen(true);
+                          setDeleteProfileName(tableMeta.rowData[1]);
+                          setDeleteProfileId(tableMeta.rowData[0]);
+                        }}
+                      />
+                    </Tooltip>
+                  </>
+                ) : (
+                  <>
+                    <Button
+                      color="green.500"
+                      onClick={() => {
+                        setIsApproveModalOpen(true);
+                        setApproveProfileName(tableMeta.rowData[1]);
+                        setApproveProfileId(tableMeta.rowData[0]);
+                      }}
+                    >
+                      Approve
+                    </Button>
+                    <Button
+                      color="red.500"
+                      onClick={() => {
+                        setIsDeleteModalOpen(true);
+                        setDeleteProfileName(tableMeta.rowData[1]);
+                        setDeleteProfileId(tableMeta.rowData[0]);
+                      }}
+                    >
+                      Reject
```

Comment:
So rejection doesn't actually mean deletion, sorry this should've been a lot more clear. The expected behaviour for rejection is to hit a backend route and send an email to the creator. There's no real copy defined for the email so I'd just say make it:
```
Hello {name},

Unfortunately, your creator profile for the Canadian Children's Book Centre creator directory has been declined. We encourage you to check over your profile again and contact us if further assistance is required.

Thanks!
CCBC
```
Their name and email should be a part of the creator model. If they're not populated yet, just don't send an email.